### DISCUSSION TITLE
Merged changes to github pages repo don't seem to be updating on the site

### DISCUSSION THREAD

**Discussion by alexander-morris** [community/community#22606] [Other Feature Feedback, Questions, & Ideas]
<p>I just merged from our develop branch to our master branch (<a href="https://github.com/BlockchainInstituteChi/www" rel="nofollow noopener">https://github.com/BlockchainInstituteChi/www</a>) and it doesn’t seem to have resulted in an update to the github pages. I’ve noticed there’s sometimes a delay but it’s been a few hours with no changes. </p>

**Comment by BaconBoi95** [comment, marked as answer]
<p>I apologize but currently, GitHub is experiencing FATAL mainframe failures and the administrators are working around the clock to fix it,</p>
<p>STATUS : <a href="http://status.github.com" rel="nofollow noopener">Status</a></p>

**Comment by BaconBoi95** [comment]
<p>Should be working again: Your webpage should have been deployed, webhooks are also now online</p>
<p>PS: A fix for this is being deployed</p>
<p>PSS: Its not just you, I can not even get to my account page </p>

**Comment by BaconBoi95** [comment]
<p>The admins are almost done reviewing the backlogs and will be recovering fully shortly</p>
<p>Stay Tuned,</p>
<p>End3rYT</p>

**Comment by alexander-morris** [comment]
<p>Thanks - appreciate the play by play. Will stay posted to the status updates. </p>

**Comment by lee-dohm** [comment]
<p>Hey <a class="mention" href="/end3ryt">@end3ryt</a>, your messages in this topic are written as if you are speaking on behalf of GitHub as a member of its staff. We understand that this is probably not your intention, so would you mind editing your posts?</p>
